navigation, welcome page and footer enhancements (#1)
Kuadrant/docs.kuadrant.io: docs/index.md

# Overview

Kuadrant brings together [Gateway API](https://gateway-api.sigs.k8s.io/) and [Open Cluster Management](https://open-cluster-management.io/) to help you scale, load-balance and secure your Ingress Gateways as a key part of your application connectivity, in the single or multi-cluster environment.

## Single-cluster

> Kuadrant can be used to manage ingress gateways based on [Gateway API](https://gateway-api.sigs.k8s.io/)[^1] for policy enforcement (rate limit and auth) in a Kuberentes cluster.

Reviewer: ```suggestion
Kuadrant can be used to protect ingress gateways based on [Gateway API](https://gateway-api.sigs.k8s.io/)[^1] with policy enforcement (rate limit and auth) in a Kuberentes cluster.
```

[^1]: <b>Supported implementations:</b> <a href="https://istio.io/">Istio</a>, <a href="https://www.redhat.com/en/technologies/cloud-computing/openshift/what-is-openshift-service-mesh">OpenShift Service Mesh</a>.</sup>

<details>
  <summary>Topology</summary>

  <img src="assets/images/architecture-single-cluster.png" alt="Single cluster architecture"/>
</details>

## Multi-cluster

You can also leverage Kuadrant for the multi-cluster environment[^2] to manage your geo-locations, global DNS, and load-balancing, and ensure gateway and policy consistency across clusters.

> 

Reviewer: ```suggestion
You can also leverage Kuadrant in a multi-cluster environment[^2] to manage DNS based north-south connectivity that can provide global load balancing that is underpinned by your cluster topology. To ensure gateway and policy consistency across clusters around important areas such as TLS and application health.
```

[^2]: Based on <a href="https://open-cluster-management.io/">Open Cluster Management</a>.

<details>
  <summary>Topology</summary>

>   <img src="assets/images/architecture-multi-cluster.png" alt="Multi cluster architecture"/>

Author: Please @maleck13 @jasonmadigan, I could use a couple extra pair of eyes to review this diagram.

Reviewer: I think this needs a few tweaks.  There is not RLP or AuthPolicy at the hub level

Reviewer: I think the  main change I would make is to remove RLP and AuthPolicy from the hub and move the DNS and TLS policy up to where RLP is targeting the gateway in the hub. Otherwise it looks pretty good

Author: Thank you @maleck13! I'll make the changes.

Author: Done.

Reviewer: Thanks @guicassolato 
just one more suggestion, the `clusterset` is kinda hanging there on its own. I think adding line to the ns with `bound` would help understand why it there
